filer/fields/multistorage_file.py

```diff
@@ -40,6 +41,32 @@ def generate_filename_multistorage(instance, filename):
         return upload_to
 
 
+class MultiStorageFileDescriptor(FileDescriptor):
+    """
+    This is rather similar to Django's ImageFileDescriptor.
+    It calls <field name>_data_changed on model instance when new
+    value is set. The callback is suposed to update fields which
+    are related to file data (like size, checksum, etc.).
+    When this is called from model __init__ (prev_assigned=False),
+    it does nothing because related fields might not have values yet.
+    In such case data_changed callback should be called at the end of model __init__
+    (File.__init__ in this case).
+    """
+    def __set__(self, instance, value):
+        prev_assigned = self.field.name in instance.__dict__
```

Comment:
@czpython I'm not sure if this is redundant because ```previous_file``` would be ```None``` both if it is not yet assigned and if it is assigned ```None```.  ```prev_assigned``` on the other hand would be true if field was assigned ```None```.